manim/mobject/geometry/line.py

```diff
@@ -27,13 +26,19 @@
 from manim.mobject.opengl.opengl_compatibility import ConvertToOpenGL
 from manim.mobject.opengl.opengl_mobject import OpenGLMobject
 from manim.mobject.types.vectorized_mobject import DashedVMobject, VGroup, VMobject
-from manim.utils.color import *
-from manim.utils.color import Colors
+from manim.utils.color import WHITE, ParsableManimColor
 from manim.utils.space_ops import angle_of_vector, line_intersection, normalize
 
 
 class Line(TipableVMobject):
-    def __init__(self, start=LEFT, end=RIGHT, buff=0, path_arc=None, **kwargs):
+    def __init__(
+        self,
+        start: Sequence[float] = LEFT,
+        end: Sequence[float] = RIGHT,
+        buff: float = 0,
+        path_arc: float | None = None,
+        **kwargs,
+    ):
```

Comment:
@alembcke I think it doesn't hurt to type `**kwargs` as `Any` and to add the `None` return type to the `__init__` method

Reply:
@Viicos thanks for the information.  I pushed a commit with that, as well as the return type `-> Self` for functions that return `self`.  Any function that did not return anything I put `-> None`, is that correct?  If the way `line.py` looks now, with all parameters and returns being typed in this way, is how a fully typed file is supposed to look, then I will do this for more files.

Reply:
Yes correct, if a function does not return anything (such as `__init__`), it returns `None` implicitly. `-> Self` is also the way to go for methods returning `self`, however we'll have to wait for the next `mypy` release to have it supported (`Self` was introduced in python3.11). I think it's fine to use it now, as a new `mypy` release will probably be available until this gets merged, and we could easily revert it if not.

Do note that you are missing the `Self` import from `typing_extensions`. I'll take a look at `line.py`, so that you'll be able to see if everything's good :)